src/helpers/isCondition/isValidAccountTokenValue.ts

```diff
@@ -3,12 +3,17 @@ import BigNumber from 'bignumber.js';
 import { LOW_LIQUIDITY_DISPLAY_TRESHOLD } from 'appConstants';
 import { TokenType } from 'types';
 
-export const isValidAccountTokenValue = (token: TokenType) => {
-  return Boolean(
+export const isValidAccountTokenValue = (token?: TokenType) => {
+  if (!token) {
+    return true;
+  }
+
+  const hasValidDisplayValue =
     token.valueUsd &&
-      (!token.isLowLiquidity ||
-        new BigNumber(token.valueUsd).isLessThan(
-          LOW_LIQUIDITY_DISPLAY_TRESHOLD
-        ))
+    token.isLowLiquidity === undefined &&
+    new BigNumber(token.valueUsd).isLessThan(LOW_LIQUIDITY_DISPLAY_TRESHOLD);
+
+  return Boolean(
+    (token.valueUsd && !token.isLowLiquidity) || hasValidDisplayValue
```

Comment:
Critical logic bug: threshold check is unreachable.

The condition !token.isLowLiquidity evaluates to true when isLowLiquidity is undefined (since !undefined === true in JavaScript). This causes the function to return true for any token where valueUsd exists and isLowLiquidity === undefined, regardless of whether valueUsd is below the threshold.

The hasValidDisplayValue condition becomes unreachable because the first part of the OR expression (token.valueUsd && !token.isLowLiquidity) already returns true when isLowLiquidity === undefined.

Apply this diff to fix the logic:

   ```return Boolean(
-    (token.valueUsd && !token.isLowLiquidity) || hasValidDisplayValue
+    (token.valueUsd && token.isLowLiquidity === false) || hasValidDisplayValue
   );```